tests/monitors/filesystems/filesystems_test.py

```diff
@@ -87,3 +91,28 @@ def test_filesystems_fstype_filter():
         ), "timed out waiting for metrics and/or dimensions!"
         assert ensure_never(lambda: has_any_metric_or_dim(agent.fake_services, expected_metrics, []))
         assert not has_log_message(agent.output.lower(), "error"), "error found in agent output!"
+
+
+def test_filesystems_extra_metrics():
+    an_extra_metric = "percent_inodes.used"
+    expected_metrics = METADATA.included_metrics | {an_extra_metric}
+    agent_config = f"""

```

Comment:
Bad line returns.

Reply:
They are still showing up.

Reply:
I don't know why this keeps happening!

Reply:
Try removing the line breaks and pressing enter again.